src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages_de.properties

```diff
@@ -3,6 +3,7 @@ assignment.inner.avoid=Innere Zuweisungen sollten vermieden werden.
 avoid.clone.method=Die Methode clone() sollte vermieden werden.
 avoid.double.brace.init=Vermeiden Sie doppelte geschweifte Klammern.
 avoid.finalizer.method=Die Methode finalize() sollte vermieden werden.
+constructors.declaration.grouping=All overloaded constructors should be grouped together. Previous constructor was at line number ''{0}''.
```

Comment:
I have not translated the error message into different languages. Because the translated messages are in some kind of encoded format and I'm not so sure how that works.

Reply:
>All overloaded constructors should be grouped together. Previous constructor was at line number ''{0}''.

make message to be `All overloaded constructors should be grouped together. Previous constructor was at line number '5'.`and use Google translator. and use all wording from Google in other languages properties. Community will fix wording eventually.

Reply:
Isn't the new message you suggested is same as the old message? Expect the `'5'`, which I believe that would be a hardcode value instead of a placeholder? 

Btw I've translated the messages into different languages as you asked. @romani

Reply:
Constructors are "overloaded" by definition, we don't need to include this term in the message at all.

Reply:
Yes, I've removed the term "overloaded" from everywhere including the error message. Here's the new error message:

https://github.com/checkstyle/checkstyle/blob/2c8f77f847bc4b8b71910ff62b61ebab5a8a2fb9/src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties#L6

I've also updated error message in other languages too.